controllers/customer-admin-controller/.gitignore

```diff
@@ -0,0 +1,77 @@
+# Temporary Build Files
```

Comment:
Since this is an image just like sync, I would try to follow the current structure we use in the repo, that is the entrypoint of the binary is found under `cmd` and its libraries under `pkg`.

Reply:
Ok, i'll attempt to move it under our current pattern of sync.

Reply:
I attempted to do this like the sync pod but ran into some errors while attempting to merge vendor directories.  The operator-sdk relies on newer libraries than we are allowing in our glide.yaml file.  I think it relies on 1.11.2 libraries and we are locking to 1.10.6.

I probably need more assistance if we attempt to place this controller in our code base.